packages/files-ui/src/Contexts/FilesContext.tsx

```diff
@@ -583,6 +621,104 @@ const FilesProvider = ({ children }: FilesContextProps) => {
         .catch(console.error)
     }, [filesApiClient, encryptForPublicKey, publicKey, refreshBuckets])
 
+  const transferFileBetweenBuckets = useCallback(async (
+    sourceBucketId: string,
+    sourceFile: FileSystemItem,
+    path: string,
+    destinationBucket: BucketKeyPermission,
+    keepOriginal = false
```

Comment:
We should default to something safe
```suggestion
    keepOriginal = true
```